ddh-intro.md

```diff
@@ -7,16 +7,18 @@
 [Instant Answers](https://duck.co/ia) are created and maintained by a community of open source developers around the world so that millions of searches on DuckDuckGo (like [who's in space?](https://duckduckgo.com/?q=who%27s+in+space%3F&ia=answer) and [tennis cheatsheet](https://duckduckgo.com/?q=tennis+cheatsheet&ia=cheatsheet&iax=1)) show a perfect result, above any ads or regular links. Just like the Wikipedia community creates articles for topics -- the DuckDuckHack community creates Instant Answers for searches!
 
 1. Use these docs and our help to improve search results for the topics you care about.
-2. Join our [dev list](https://www.listbox.com/subscribe/?list_id=197814) to keep in touch with all the latest news and DuckDuckHack events.
-3. Become a leader and all-star contributor helping to change the world of search through community!
+2. Sign up for our [dev list](https://www.listbox.com/subscribe/?list_id=197814) to keep in touch with all the latest news and DuckDuckHack events.
```

Comment:
Could we say something like `to get the latest DuckDuckHack news, contribution suggestions and programming tips` here?